src/main/java/net/mcreator/element/ModElementTypeRegistry.java

```diff
@@ -51,7 +51,7 @@
 		put(ModElementType.GUI, new ModTypeRegistration<>('g', 19, CustomGUIGUI::new, GUI.class));
 		put(ModElementType.OVERLAY, new ModTypeRegistration<>('v', 9, OverlayGUI::new, Overlay.class));
 		put(ModElementType.PROCEDURE, new ModTypeRegistration<>('p', 17, ProcedureGUI::new, Procedure.class));
-		put(ModElementType.POTION, new ModTypeRegistration<>('z', 23, PotionGUI::new, Potion.class));
+		put(ModElementType.POTIONEFFECT, new ModTypeRegistration<>('z', 32, PotionEffectGUI::new, PotionEffect.class));

```

Comment:
Old class names should be kept. Advancement still has `Achievement` for its two classes.